habitat/tasks/eqa/eqa.py

```diff
@@ -48,77 +45,169 @@ class EQAEpisode(NavigationEpisode):
     )
 
 
+@registry.register_sensor
 class QuestionSensor(Sensor):
-    def __init__(self, **kwargs):
+    def __init__(self, dataset, *args: Any, **kwargs: Any):
         self.uuid = "question"
         self.sensor_type = SensorTypes.TEXT
-        # TODO (maksymets) extend gym observation space for text and metadata
-        self.observation_space = spaces.Discrete(0)
+        self._dataset = dataset
+        self.observation_space = spaces.Discrete(
```

Comment:
Is this used anywhere?  If so, why is it different than `_get_observation_space`?